index.js

```diff
@@ -0,0 +1,61 @@
+class BookList {
+  constructor() {
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+    this.bookListElement = document.getElementById('bookList');
+    this.bookTitleInput = document.getElementById('bookTitle');
+    this.bookAuthorInput = document.getElementById('bookAuthor');
+    this.addButton = document.getElementById('addButton');
+
+    this.addButton.addEventListener('click', () => {
+      this.addBook();
+      this.renderBooks();
+      this.bookTitleInput.value = '';
+      this.bookAuthorInput.value = '';
+    });
+
+    this.renderBooks();
+  }
+
+  addBook() {
+    const title = this.bookTitleInput.value.trim();
+    const author = this.bookAuthorInput.value.trim();
+    if (title !== '' && author !== '') {
+      const book = { title, author };
+      this.books.push(book);
+      this.saveBooks();
+    }
+  }
+
+  removeBook(index) {
+    this.books.splice(index, 1);
+    this.saveBooks();
+  }
+
+  saveBooks() {
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
+
+  renderBooks() {
+    this.bookListElement.innerHTML = '';
+    this.books.forEach((book, index) => {
+      const li = document.createElement('li');
+      li.textContent = `${book.title} by ${book.author}`;
+
+      const removeButton = document.createElement('button');
+      removeButton.textContent = 'Remove';
+      removeButton.classList.add('remove-button');
+      removeButton.addEventListener('click', () => {
+        this.removeBook(index);
+        this.renderBooks();
+      });
+
+      li.appendChild(removeButton);
+      this.bookListElement.appendChild(li);
+    });
+  }
+}
+// eslint-disable-next-line no-new
+document.addEventListener('DOMContentLoaded', () => {
+  // eslint-disable-next-line no-new
+  new BookList();
+});
```

Comment:
- [ ] Good job removing the `eslint-disable` at the top of the file, however, you have introduced two new `eslint-disable-next-line no-new` comments. Kindly note that the suggestion by the previous [reviewer here](https://github.com/Sami-Ullah-Tufail/Awesome-books/pull/3#discussion_r1288230483) to render the book when the DOM loads, left by the previous reviewer was optional and if it is causing an error kindly fix it.